nautobot/extras/datasources/git.py

```diff
@@ -716,56 +715,41 @@ def delete_git_config_context_schemas(repository_record, job_result, preserve=()
 #
 
 
-def refresh_code_from_repository(repository_slug, consumer=None, skip_reimport=False):
+def refresh_code_from_repository(repository_slug, skip_reimport=False, should_raise=False):
```

Comment:
Can we rename this function or update the docstring to state that this is only reloading jobs code? There's a lot of code in a git repository that could potentially be refreshed.